priiloader/source/hacks.cpp

```diff
@@ -242,6 +242,18 @@ bool _processLine(system_hack& hack, std::string &line)
 			//not needed line, but we used to have it. we do nothing with it though...
 			//hack.amount = strtoul(values.front().c_str(),NULL, 10);
 		}
+		else if(type == "provide")

```

Comment:
one last note : the variables were renamed, but not these.
i'd rename them as well. (something like masterID & requiredID ).

thats my only comment. ill look into the lambda's after the merge (cause ye, i didn't use them yet, but i should've)

Reply:
"masterID" and "requiredID" would look a bit weird, being the only non-lowercase names then. What about "master" and "require"?

Reply:
true. that would look weird. 
i'd say master & requires?